modules/local/merge_beds.nf

```diff
@@ -9,6 +9,7 @@ process MERGE_BEDS {
 
     input:
     tuple val(meta), path(bed, stageAs: "?/*")
+    path(fai)
```

Comment:
```suggestion
    tuple val(meta2), path(fai)
```

?

Reply:
Eh it's a local module. I don't have meta by default on my `fai` channel